src/dropcursor.js

```diff
@@ -59,7 +59,7 @@ class DropCursorView {
       rect = {left: coords.left - this.width / 2, right: coords.left + this.width / 2, top: coords.top, bottom: coords.bottom}
     }
 
-    let parent = this.editorView.dom.offsetParent
+    let parent = this.editorView.dom.offsetParent || this.editorView.dom; // e.g. body.offsetParent is null
```

Comment:
`let parent = this.editorView.dom.offsetParent || this.editorView.dom // e.g. body.offsetParent is null`